app/controllers/users/verify_account_controller.rb

```diff
@@ -8,9 +8,9 @@ def index
 
     def create
       @verify_account_form = build_verify_account_form
+
       if @verify_account_form.submit
-        flash[:success] = t('account.index.verification.success')
-        redirect_to account_path
+        redirect_to sign_up_completed_url
```

Comment:
Yeah, on my way in to work, I realized I was wrong about this scenario not being likely. Even if the instructions didn't include a URL, the user could potentially go to the SP directly, then after they sign in on login.gov, they will be prompted to confirm their account, and at that point, the SP info will be in the session.

I think it's still worth it to have tests for both scenarios: one where the SP info is in the session, and one where it is not.

Also, I'm curious how the "URL that points back to the service provider" works. Where can I see an example of that?